02-23 19:14:21 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:204)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:391)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2843)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:128)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Unknown Source)
02-23 22:26:51 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:204)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:391)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2843)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:128)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Unknown Source)
02-23 22:27:13 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.getStatement(JdbcResultSet.java:225)
	at net.sourceforge.squirrel_sql.fw.sql.SQLUtilities.closeResultSet(SQLUtilities.java:349)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.expanders.IndexParentExpander.createChildren(IndexParentExpander.java:119)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.loadChildren(TreeLoader.java:126)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.execute(TreeLoader.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.expandNode(ObjectTree.java:570)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$1100(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController.run(ObjectTree.java:936)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.startExpandingTree(ObjectTree.java:535)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$800(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$NodeExpansionListener.treeExpanded(ObjectTree.java:879)
	at javax.swing.JTree.fireTreeExpanded(Unknown Source)
	at javax.swing.JTree.setExpandedState(Unknown Source)
	at javax.swing.JTree.expandPath(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI.toggleExpandState(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI.selectPathForEvent(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.handleSelection(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.mouseReleasedDND(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.mouseReleased(Unknown Source)
	at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
	at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at net.sourceforge.squirrel_sql.client.Main$1.dispatchEvent(Main.java:99)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
02-23 22:27:41 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.getStatement(JdbcResultSet.java:225)
	at net.sourceforge.squirrel_sql.fw.sql.SQLUtilities.closeResultSet(SQLUtilities.java:349)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.expanders.IndexParentExpander.createChildren(IndexParentExpander.java:119)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.loadChildren(TreeLoader.java:126)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.execute(TreeLoader.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.expandNode(ObjectTree.java:570)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$1100(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController.run(ObjectTree.java:936)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.startExpandingTree(ObjectTree.java:535)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$800(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$NodeExpansionListener.treeExpanded(ObjectTree.java:879)
	at javax.swing.JTree.fireTreeExpanded(Unknown Source)
	at javax.swing.JTree.setExpandedState(Unknown Source)
	at javax.swing.JTree.expandPath(Unknown Source)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:450)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$1400(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController$1.run(ObjectTree.java:945)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:367)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:332)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController.run(ObjectTree.java:941)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.startExpandingTree(ObjectTree.java:535)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.refreshTree(ObjectTree.java:362)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$600(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$3$1.run(ObjectTree.java:330)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:367)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:332)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$3.run(ObjectTree.java:326)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.refresh(ObjectTree.java:343)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTreePanel.refreshTree(ObjectTreePanel.java:648)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTreePanel$2.schemaInfoUpdated(ObjectTreePanel.java:221)
	at net.sourceforge.squirrel_sql.client.session.schemainfo.SchemaInfo$10.run(SchemaInfo.java:1510)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at net.sourceforge.squirrel_sql.client.Main$1.dispatchEvent(Main.java:99)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
02-23 22:27:41 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.getStatement(JdbcResultSet.java:225)
	at net.sourceforge.squirrel_sql.fw.sql.SQLUtilities.closeResultSet(SQLUtilities.java:349)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.expanders.IndexParentExpander.createChildren(IndexParentExpander.java:119)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.loadChildren(TreeLoader.java:126)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.execute(TreeLoader.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.expandNode(ObjectTree.java:570)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$1100(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController.run(ObjectTree.java:936)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.startExpandingTree(ObjectTree.java:535)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$800(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$NodeExpansionListener.treeExpanded(ObjectTree.java:879)
	at javax.swing.JTree.fireTreeExpanded(Unknown Source)
	at javax.swing.JTree.setExpandedState(Unknown Source)
	at javax.swing.JTree.expandPath(Unknown Source)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:450)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.restoreExpansionState(ObjectTree.java:480)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$1400(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController$1.run(ObjectTree.java:945)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:367)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:332)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController.run(ObjectTree.java:941)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.startExpandingTree(ObjectTree.java:535)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.refreshTree(ObjectTree.java:362)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$600(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$3$1.run(ObjectTree.java:330)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:367)
	at net.sourceforge.squirrel_sql.fw.gui.GUIUtils.processOnSwingEventThread(GUIUtils.java:332)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$3.run(ObjectTree.java:326)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.refresh(ObjectTree.java:343)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTreePanel.refreshTree(ObjectTreePanel.java:648)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTreePanel$2.schemaInfoUpdated(ObjectTreePanel.java:221)
	at net.sourceforge.squirrel_sql.client.session.schemainfo.SchemaInfo$10.run(SchemaInfo.java:1510)
	at java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at net.sourceforge.squirrel_sql.client.Main$1.dispatchEvent(Main.java:99)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
02-23 22:46:03 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.getStatement(JdbcResultSet.java:225)
	at org.executequery.gui.resultset.ResultSetTableModel.createTable(ResultSetTableModel.java:281)
	at org.executequery.gui.browser.DataTypesPanel.setDataTypes(DataTypesPanel.java:73)
	at org.executequery.gui.browser.HostPanel.updateDatabaseTypeInfo(HostPanel.java:232)
	at org.executequery.gui.browser.HostPanel.changePanelData(HostPanel.java:253)
	at org.executequery.gui.browser.HostPanel.access$000(HostPanel.java:53)
	at org.executequery.gui.browser.HostPanel$1.run(HostPanel.java:273)
	at org.underworldlabs.swing.GUIUtils$2.construct(GUIUtils.java:244)
	at org.underworldlabs.swing.util.SwingWorker$2.run(SwingWorker.java:130)
	at java.lang.Thread.run(Unknown Source)
02-23 22:49:42 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.getStatement(JdbcResultSet.java:225)
	at org.executequery.gui.resultset.ResultSetTableModel.createTable(ResultSetTableModel.java:281)
	at org.executequery.gui.browser.DataTypesPanel.setDataTypes(DataTypesPanel.java:73)
	at org.executequery.gui.browser.HostPanel.updateDatabaseTypeInfo(HostPanel.java:232)
	at org.executequery.gui.browser.HostPanel.changePanelData(HostPanel.java:253)
	at org.executequery.gui.browser.HostPanel.access$000(HostPanel.java:53)
	at org.executequery.gui.browser.HostPanel$1.run(HostPanel.java:273)
	at org.underworldlabs.swing.GUIUtils$2.construct(GUIUtils.java:244)
	at org.underworldlabs.swing.util.SwingWorker$2.run(SwingWorker.java:130)
	at java.lang.Thread.run(Unknown Source)
02-23 23:15:41 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Feature not supported: "clientInfoProperties" [50100-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.getUnsupportedException(DbException.java:204)
	at org.h2.message.TraceObject.unsupported(TraceObject.java:391)
	at org.h2.jdbc.JdbcDatabaseMetaData.getClientInfoProperties(JdbcDatabaseMetaData.java:2843)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.executeGetter(MetaDataDataSet.java:255)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.generateLine(MetaDataDataSet.java:222)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.load(MetaDataDataSet.java:149)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDataSet.<init>(MetaDataDataSet.java:96)
	at net.sourceforge.squirrel_sql.fw.sql.MetaDataDecoratorDataSet.<init>(MetaDataDecoratorDataSet.java:73)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.database.MetaDataTab.createDataSet(MetaDataTab.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.tabs.BaseDataSetTab$1.run(BaseDataSetTab.java:128)
	at net.sourceforge.squirrel_sql.fw.util.TaskExecuter.run(TaskExecuter.java:82)
	at java.lang.Thread.run(Unknown Source)
02-23 23:16:44 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcResultSet.checkClosed(JdbcResultSet.java:2945)
	at org.h2.jdbc.JdbcResultSet.getStatement(JdbcResultSet.java:225)
	at net.sourceforge.squirrel_sql.fw.sql.SQLUtilities.closeResultSet(SQLUtilities.java:349)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.expanders.IndexParentExpander.createChildren(IndexParentExpander.java:119)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.loadChildren(TreeLoader.java:126)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.TreeLoader.execute(TreeLoader.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.expandNode(ObjectTree.java:570)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$1100(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$ExpansionController.run(ObjectTree.java:936)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.startExpandingTree(ObjectTree.java:535)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree.access$800(ObjectTree.java:74)
	at net.sourceforge.squirrel_sql.client.session.mainpanel.objecttree.ObjectTree$NodeExpansionListener.treeExpanded(ObjectTree.java:879)
	at javax.swing.JTree.fireTreeExpanded(Unknown Source)
	at javax.swing.JTree.setExpandedState(Unknown Source)
	at javax.swing.JTree.expandPath(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI.toggleExpandState(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI.handleExpandControlClick(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI.checkForClickInExpandControl(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.handleSelection(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.mousePressedDND(Unknown Source)
	at javax.swing.plaf.basic.BasicTreeUI$Handler.mousePressed(Unknown Source)
	at java.awt.AWTEventMulticaster.mousePressed(Unknown Source)
	at java.awt.AWTEventMulticaster.mousePressed(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$200(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at net.sourceforge.squirrel_sql.client.Main$1.dispatchEvent(Main.java:99)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
02-24 12:38:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 12:38:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 12:46:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
ALTER TABLE PUBLIC.ACTIVITE ALTER COLUMN IDACTIVITE INTEGER NOT NULL IDENTITY [90017-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:103)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:432)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:368)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:214)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:138)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.onseven.dbvis.g.B.B.ā(Z:1910)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 12:47:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
ALTER TABLE PUBLIC.ACTIVITE ALTER COLUMN IDACTIVITE INTEGER NOT NULL IDENTITY [90017-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:103)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:432)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:368)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:214)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:138)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.onseven.dbvis.g.B.B.ā(Z:1910)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 12:47:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
ALTER TABLE PUBLIC.ACTIVITE ALTER COLUMN IDACTIVITE INTEGER NOT NULL IDENTITY [90017-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:103)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:432)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:368)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:214)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:138)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.onseven.dbvis.g.B.B.ā(Z:1910)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 12:48:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Attempt to define a second primary key; SQL statement:
ALTER TABLE PUBLIC.ACTIVITE ALTER COLUMN IDACTIVITE INTEGER NOT NULL IDENTITY [90017-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:103)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:68)
	at org.h2.command.ddl.AlterTableAlterColumn.execute(AlterTableAlterColumn.java:432)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:368)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:214)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:138)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.onseven.dbvis.g.B.B.ā(Z:1910)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 12:56:45 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE PUBLIC.MEMBRE ALTER COLUMN DATENAISS TIMESTAMP DEFAULT 1900-01-01 00[*]:00:00"; SQL statement:
ALTER TABLE PUBLIC.MEMBRE ALTER COLUMN DATENAISS TIMESTAMP DEFAULT 1900-01-01 00:00:00 [42000-167]
02-24 13:04:52 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE PUBLIC.ACTIVITE ADD CONSTRAINT ACTIVITE_FK1 FOREIGN KEY (IDFORMATION) REFERENCES PUBLIC.ACTIVITE ()[*] ON DELETE RESTRICT ON UPDATE SET NULL "; expected "identifier"; SQL statement:
ALTER TABLE PUBLIC.ACTIVITE ADD CONSTRAINT ACTIVITE_fk1 FOREIGN KEY (IDFORMATION) REFERENCES PUBLIC.ACTIVITE () ON DELETE RESTRICT ON UPDATE SET NULL [42001-167]
02-24 13:09:13 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 13:09:13 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 13:09:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 13:09:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 13:09:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 13:09:13 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 13:18:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT @@[*]IDENTITY"; expected "identifier"; SQL statement:
SELECT @@IDENTITY [42001-170]
02-24 19:18:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 19:18:12 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 19:18:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 19:18:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 19:20:22 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
02-24 19:20:22 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: The object is already closed [90007-167]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1383)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1361)
	at org.h2.jdbc.JdbcConnection.getAutoCommit(JdbcConnection.java:423)
	at com.onseven.dbvis.g.B.C.ā(Z:3551)
	at com.onseven.dbvis.g.B.F$A.call(Z:1474)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
03-03 12:01:32 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSCRIPTION[*] SELECT * FROM ACTIVITE"; expected "INSERT, {"; SQL statement:
INSCRIPTION SELECT * FROM ACTIVITE [42001-170]
03-03 12:12:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "JAVA[*] -CP H2*.JAR ORG.H2.TOOLS.CHANGEFILEENCRYPTION -DIR D:\DOCUMENTS\NETBEANSPROJECTS\MIVIAPP\DB -DB MIVIDB -CIPHER AES -ENCRYPT TEROKNOR "; SQL statement:
java -cp h2*.jar org.h2.tools.ChangeFileEncryption -dir D:\Documents\NetBeansProjects\MiviApp\db -db mividb -cipher AES -encrypt teroknor [42000-170]
03-03 15:57:18 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Parameter "#1" is not set; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, sum(paie.montant)
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre = ? and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation and paie.idActivite = ins.idActivite and paie.idMembre = ins.idMembre
  group by ins.idActivite order by form.intitule [90012-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.Parameter.checkSet(Parameter.java:73)
	at org.h2.command.Prepared.checkParameters(Prepared.java:163)
	at org.h2.command.CommandContainer.query(CommandContainer.java:85)
	at org.h2.command.Command.executeQuery(Command.java:191)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:173)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:08:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT FORM.INTITULE, ACT.PROMOTION, ACT.DATEDEB, INS.TARIFSPECIAL, ACT.PRIX, INS.CERTIFIE, SUM(PAIE.MONTANT)
  FROM INSCRIPTION INS, ACTIVITE ACT, FORMATION FORM, PAIEMENT PAIE
  WHERE INS.IDMEMBRE =  29 AND INS.IDACTIVITE = ACT.IDACTIVITE AND
  ACT.IDFORMATION = FORM.IDFORMATION AND 
  CASE(SELECT COUNT(IDMEMBRE) FROM PAIEMENT WHERE PAIEMENT.IDMEMBRE = INS.IDMEMBRE AND PAIEMENT.IDACTIVITE = INS.IDACTIVITE)
    WHEN >[*]0 THEN PAIE.IDACTIVITE = INS.IDACTIVITE AND PAIE.IDMEMBRE = INS.IDMEMBRE END
  GROUP BY INS.IDACTIVITE ORDER BY FORM.INTITULE "; expected "NOT, EXISTS"; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, sum(paie.montant)
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation and 
  case(select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite)
    when >0 then paie.idActivite = ins.idActivite and paie.idMembre = ins.idMembre end
  group by ins.idActivite order by form.intitule [42001-170]
03-03 16:11:26 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT FORM.INTITULE, ACT.PROMOTION, ACT.DATEDEB, INS.TARIFSPECIAL, ACT.PRIX, INS.CERTIFIE, SUM(PAIE.MONTANT)
  FROM INSCRIPTION INS, ACTIVITE ACT, FORMATION FORM, PAIEMENT PAIE
  WHERE INS.IDMEMBRE =  29 AND INS.IDACTIVITE = ACT.IDACTIVITE AND
  ACT.IDFORMATION = FORM.IDFORMATION AND 
  CASE
    WHEN (SELECT COUNT(IDMEMBRE) FROM PAIEMENT WHERE PAIEMENT.IDMEMBRE = INS.IDMEMBRE AND PAIEMENT.IDACTIVITE = INS.IDACTIVITE)>0 THEN PAIE.IDACTIVITE = INS.IDACTIVITE AND PAIE.IDMEMBRE = INS.IDMEMBRE 
 ELSE END
  GROUP[*] BY INS.IDACTIVITE ORDER BY FORM.INTITULE "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, END"; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, sum(paie.montant)
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation and 
  case
    when (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite)>0 then paie.idActivite = ins.idActivite and paie.idMembre = ins.idMembre 
 else end
  group by ins.idActivite order by form.intitule [42001-170]
03-03 16:14:54 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as NbPaiement
 case when NbPaiement > 0 then sum(paie.montant) else 0 end as Paye
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite order by form.intitule [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:14:58 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as NbPaiement
 case when NbPaiement > 0 then sum(paie.montant) else 0 end as Paye 
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite order by form.intitule [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:15:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as NbPaiement
 case paye when NbPaiement > 0 then sum(paie.montant) else 0 end
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite order by form.intitule [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:16:11 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as NbPaiement
 case when NbPaiement > 0 then sum(paie.montant) else 0 end
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite order by form.intitule [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:17:06 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as nbpaiements
 case when nbpaiements > 0 then sum(paie.montant) else 0 end as paye
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite order by form.intitule [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:17:06 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as nbpaiements
 case when nbpaiements > 0 then sum(paie.montant) else 0 end as paye
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite order by form.intitule [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:17:10 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as nbpaiements
 case when nbpaiements > 0 then sum(paie.montant) else 0 end as paye 
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite order by form.intitule [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:17:15 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "FORM.INTITULE" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as nbpaiements
 case when nbpaiements > 0 then sum(paie.montant) else 0 end as paye 
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:17:27 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "ACT.PROMOTION" not found; SQL statement:
select act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as nbpaiements
 case when nbpaiements > 0 then sum(paie.montant) else 0 end as paye 
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:17:48 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "NBPAIEMENTS" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) as nbpaiements,
 case when nbpaiements > 0 then sum(paie.montant) else 0 end as paye 
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
  group by ins.idActivite [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.expression.Comparison.optimize(Comparison.java:159)
	at org.h2.expression.Function.optimize(Function.java:1767)
	at org.h2.expression.Alias.optimize(Alias.java:47)
	at org.h2.command.dml.Select.prepare(Select.java:799)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:19:42 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "ACT.IDFORMA" not found; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, 
 case when (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) > 0 then           sum(paie.montant) else 0 end as paye 
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idForma tion = form.idFormation 
case when (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) > 0 then 
paie.idMembre = ins.idMembre and paie.idActivite = ins.idActivite end
  group by ins.idActivite [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.expression.ConditionAndOr.optimize(ConditionAndOr.java:127)
	at org.h2.command.dml.Select.prepare(Select.java:802)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-03 16:19:52 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT FORM.INTITULE, ACT.PROMOTION, ACT.DATEDEB, INS.TARIFSPECIAL, ACT.PRIX, INS.CERTIFIE, 
 CASE WHEN (SELECT COUNT(IDMEMBRE) FROM PAIEMENT WHERE PAIEMENT.IDMEMBRE = INS.IDMEMBRE AND PAIEMENT.IDACTIVITE = INS.IDACTIVITE) > 0 THEN           SUM(PAIE.MONTANT) ELSE 0 END AS PAYE 
  FROM INSCRIPTION INS, ACTIVITE ACT, FORMATION FORM, PAIEMENT PAIE
  WHERE INS.IDMEMBRE =  29 AND INS.IDACTIVITE = ACT.IDACTIVITE AND
  ACT.IDFORMATION = FORM.IDFORMATION 
CASE[*] WHEN (SELECT COUNT(IDMEMBRE) FROM PAIEMENT WHERE PAIEMENT.IDMEMBRE = INS.IDMEMBRE AND PAIEMENT.IDACTIVITE = INS.IDACTIVITE) > 0 THEN 
PAIE.IDMEMBRE = INS.IDMEMBRE AND PAIE.IDACTIVITE = INS.IDACTIVITE END
  GROUP BY INS.IDACTIVITE "; SQL statement:
select form.intitule, act.promotion, act.dateDeb, ins.tarifSpecial, act.prix, ins.certifie, 
 case when (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) > 0 then           sum(paie.montant) else 0 end as paye 
  from inscription ins, activite act, formation form, paiement paie
  where ins.idMembre =  29 and ins.idActivite = act.idActivite and
  act.idFormation = form.idFormation 
case when (select count(idMembre) from paiement where paiement.idMembre = ins.idMembre and paiement.idActivite = ins.idActivite) > 0 then 
paie.idMembre = ins.idMembre and paie.idActivite = ins.idActivite end
  group by ins.idActivite [42000-170]
03-03 16:22:45 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "1" for parameter "parameterIndex" [90008-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1284)
	at org.h2.jdbc.JdbcPreparedStatement.setInt(JdbcPreparedStatement.java:321)
	at dataAccessPackage.AccessInscriptionDB.listInsMembre(AccessInscriptionDB.java:160)
	at businessPackage.InscriptionManager.listInsMembre(InscriptionManager.java:43)
	at controllerPackage.ApplicationController.listInsMembre(ApplicationController.java:122)
	at viewPackage.PanelMembre.UpdateHistoryMembre(PanelMembre.java:491)
	at viewPackage.PanelMembre.UpdateMembre(PanelMembre.java:416)
	at viewPackage.PanelMembre.access$100(PanelMembre.java:47)
	at viewPackage.PanelMembre$1.valueChanged(PanelMembre.java:89)
	at javax.swing.JList.fireSelectionValueChanged(JList.java:1765)
	at javax.swing.JList$ListSelectionHandler.valueChanged(JList.java:1779)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:167)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:147)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:194)
	at javax.swing.DefaultListSelectionModel.changeSelection(DefaultListSelectionModel.java:388)
	at javax.swing.DefaultListSelectionModel.changeSelection(DefaultListSelectionModel.java:398)
	at javax.swing.DefaultListSelectionModel.setSelectionInterval(DefaultListSelectionModel.java:442)
	at javax.swing.JList.setSelectionInterval(JList.java:2035)
	at javax.swing.plaf.basic.BasicListUI$Handler.adjustSelection(BasicListUI.java:2727)
	at javax.swing.plaf.basic.BasicListUI$Handler.mousePressed(BasicListUI.java:2683)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:263)
	at java.awt.Component.processMouseEvent(Component.java:6294)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3275)
	at java.awt.Component.processEvent(Component.java:6062)
	at java.awt.Container.processEvent(Container.java:2039)
	at java.awt.Component.dispatchEventImpl(Component.java:4660)
	at java.awt.Container.dispatchEventImpl(Container.java:2097)
	at java.awt.Component.dispatchEvent(Component.java:4488)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4575)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4233)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4166)
	at java.awt.Container.dispatchEventImpl(Container.java:2083)
	at java.awt.Window.dispatchEventImpl(Window.java:2489)
	at java.awt.Component.dispatchEvent(Component.java:4488)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:668)
	at java.awt.EventQueue.access$400(EventQueue.java:81)
	at java.awt.EventQueue$2.run(EventQueue.java:627)
	at java.awt.EventQueue$2.run(EventQueue.java:625)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:641)
	at java.awt.EventQueue$3.run(EventQueue.java:639)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:638)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
03-03 18:22:36 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "4" for parameter "columnIndex" [90008-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:2934)
	at org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:2962)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:307)
	at dataAccessPackage.AccessInscriptionDB.listInscription(AccessInscriptionDB.java:58)
	at businessPackage.InscriptionManager.listInscription(InscriptionManager.java:27)
	at controllerPackage.ApplicationController.listInscription(ApplicationController.java:106)
	at viewPackage.PanelInscription.UpdateListInscription(PanelInscription.java:682)
	at viewPackage.PanelInscription.access$4900(PanelInscription.java:57)
	at viewPackage.PanelInscription$ListListener.valueChanged(PanelInscription.java:640)
	at javax.swing.JList.fireSelectionValueChanged(JList.java:1765)
	at javax.swing.JList$ListSelectionHandler.valueChanged(JList.java:1779)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:167)
	at javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:137)
	at javax.swing.DefaultListSelectionModel.setValueIsAdjusting(DefaultListSelectionModel.java:668)
	at javax.swing.JList.setValueIsAdjusting(JList.java:2110)
	at javax.swing.plaf.basic.BasicListUI$Handler.mouseReleased(BasicListUI.java:2784)
	at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:273)
	at java.awt.Component.processMouseEvent(Component.java:6297)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3275)
	at java.awt.Component.processEvent(Component.java:6062)
	at java.awt.Container.processEvent(Container.java:2039)
	at java.awt.Component.dispatchEventImpl(Component.java:4660)
	at java.awt.Container.dispatchEventImpl(Container.java:2097)
	at java.awt.Component.dispatchEvent(Component.java:4488)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4575)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4236)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4166)
	at java.awt.Container.dispatchEventImpl(Container.java:2083)
	at java.awt.Window.dispatchEventImpl(Window.java:2489)
	at java.awt.Component.dispatchEvent(Component.java:4488)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:668)
	at java.awt.EventQueue.access$400(EventQueue.java:81)
	at java.awt.EventQueue$2.run(EventQueue.java:627)
	at java.awt.EventQueue$2.run(EventQueue.java:625)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:641)
	at java.awt.EventQueue$3.run(EventQueue.java:639)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:638)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
03-05 22:06:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE MEMBRE SET COLUMN[*] IDJORIS TYPE STRING "; expected "REFERENTIAL_INTEGRITY"; SQL statement:
alter table membre set column idjoris type String [42001-170]
03-05 22:07:46 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER TABLE MEMBRE MODIFY[*] IDJORIS STRING "; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
alter table membre modify idjoris String [42001-170]
03-05 22:48:13 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "IDJORIS"; expected "INSERT, {"; SQL statement:
IDJORIS [42001-170]
03-06 13:24:23 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Invalid value "12" for parameter "parameterIndex" [90008-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.getInvalidValueException(DbException.java:215)
	at org.h2.jdbc.JdbcPreparedStatement.setParameter(JdbcPreparedStatement.java:1284)
	at org.h2.jdbc.JdbcPreparedStatement.setLong(JdbcPreparedStatement.java:561)
	at dataAccess.AccessMembre.ImportMembre(AccessMembre.java:104)
	at controller.ExportAccessToH2.main(ExportAccessToH2.java:34)
03-06 13:29:54 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "TRUNCATE MEMBRE[*] "; expected "TABLE"; SQL statement:
truncate membre [42001-170]
03-06 14:14:34 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "BIGINT" not found; SQL statement:
ALTER TABLE membre ALTER COLUMN idmembre  RESTART WITH bigint [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.AlterSequence.update(AlterSequence.java:67)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:14:43 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "BIGINT" not found; SQL statement:
ALTER TABLE membre ALTER COLUMN idmembre  RESTART WITH bigint [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.AlterSequence.update(AlterSequence.java:67)
	at org.h2.command.CommandContainer.update(CommandContainer.java:75)
	at org.h2.command.Command.executeUpdate(Command.java:230)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:177)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:15:49 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ALTER SEQUENCE "; expected "identifier"; SQL statement:
alter sequence [42001-170]
03-06 14:16:07 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "SEQUENCE" not found; SQL statement:
alter table membre alter sequence: [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:605)
	at org.h2.command.Parser.parseAlterTable(Parser.java:4884)
	at org.h2.command.Parser.parseAlter(Parser.java:4314)
	at org.h2.command.Parser.parsePrepared(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.parse(Parser.java:250)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:16:09 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "SEQUENCE" not found; SQL statement:
alter table membre alter sequence [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.table.Table.getColumn(Table.java:605)
	at org.h2.command.Parser.parseAlterTable(Parser.java:4884)
	at org.h2.command.Parser.parseAlter(Parser.java:4314)
	at org.h2.command.Parser.parsePrepared(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:278)
	at org.h2.command.Parser.parse(Parser.java:250)
	at org.h2.command.Parser.prepareCommand(Parser.java:217)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:41:29 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES (PRATICIEN EN[*] COACHING MENTAL) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values (Praticien en Coaching Mental) [42001-170]
03-06 14:41:29 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES (PRATICIEN EN[*] COACHING MENTAL) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values (Praticien en Coaching Mental) [42001-170]
03-06 14:41:29 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES (PRATICIEN EN[*] COACHING MENTAL) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values (Praticien en Coaching Mental) [42001-170]
03-06 14:41:29 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES (PRATICIEN EN[*] COACHING MENTAL) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values (Praticien en Coaching Mental) [42001-170]
03-06 14:41:29 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES (PRATICIEN EN[*] COACHING MENTAL) "; expected "., (, [, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values (Praticien en Coaching Mental) [42001-170]
03-06 14:46:11 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "Praticien en Coaching Mental" not found; SQL statement:
insert into formation ( intitule ) values ("Praticien en Coaching Mental") [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:46:11 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "Maître Praticien en Coaching Mental" not found; SQL statement:
insert into formation ( intitule ) values ("Maître Praticien en Coaching Mental") [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:46:11 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "Praticien en Hypnose Ericksonienne" not found; SQL statement:
insert into formation ( intitule ) values ("Praticien en Hypnose Ericksonienne") [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:53:04 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES ('Académie d'ÉTÉ[*]');
insert into formation ( intitule ) values ('ACADÉMIE D’HIVER');
insert into formation ( intitule ) values ('L’ART DES MÉTAPHORES DANS LE COACHING MENTAL');
insert into formation ( intitule ) values ('AUTO-HYPNOSE');
insert into formation ( intitule ) values ('RENFORCER LA CONFIANCE EN SOI');
insert into formation ( intitule ) values ('CHANGER VOS CROYANCES LIMITANTES AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('COMMENT LANCER VOTRE ACTIVITÉ DE COACH MENTAL');
insert into formation ( intitule ) values ('APPRENDRE LA SOPHROLOGIE');
insert into formation ( intitule ) values ('COMPRENDRE ET INTÉGRER LES EMOTIONS');
insert into formation ( intitule ) values ('MAIGRIR AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('TECHNIQUES DE COACHING MENTAL POUR LE SPORT');
insert into formation ( intitule ) values ('GÉRER LE STRESS AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('CHANGER SA RELATION À L'argent avec le Coaching Mental') "; expected "[, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values ('Académie d'Été');
insert into formation ( intitule ) values ('Académie d’Hiver');
insert into formation ( intitule ) values ('L’Art des Métaphores dans le Coaching Mental');
insert into formation ( intitule ) values ('Auto-hypnose');
insert into formation ( intitule ) values ('Renforcer la Confiance en Soi');
insert into formation ( intitule ) values ('Changer vos Croyances Limitantes avec le Coaching Mental');
insert into formation ( intitule ) values ('Comment Lancer votre Activité de Coach Mental');
insert into formation ( intitule ) values ('Apprendre la Sophrologie');
insert into formation ( intitule ) values ('Comprendre et Intégrer les Emotions');
insert into formation ( intitule ) values ('Maigrir avec le Coaching Mental');
insert into formation ( intitule ) values ('Techniques de Coaching Mental pour le Sport');
insert into formation ( intitule ) values ('Gérer le Stress avec le Coaching Mental');
insert into formation ( intitule ) values ('Changer sa relation à l'argent avec le Coaching Mental') [42001-170]
03-06 14:54:39 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES ('Académie d\'ÉTÉ[*]');
insert into formation ( intitule ) values ('ACADÉMIE D’HIVER');
insert into formation ( intitule ) values ('L’ART DES MÉTAPHORES DANS LE COACHING MENTAL');
insert into formation ( intitule ) values ('AUTO-HYPNOSE');
insert into formation ( intitule ) values ('RENFORCER LA CONFIANCE EN SOI');
insert into formation ( intitule ) values ('CHANGER VOS CROYANCES LIMITANTES AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('COMMENT LANCER VOTRE ACTIVITÉ DE COACH MENTAL');
insert into formation ( intitule ) values ('APPRENDRE LA SOPHROLOGIE');
insert into formation ( intitule ) values ('COMPRENDRE ET INTÉGRER LES EMOTIONS');
insert into formation ( intitule ) values ('MAIGRIR AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('TECHNIQUES DE COACHING MENTAL POUR LE SPORT');
insert into formation ( intitule ) values ('GÉRER LE STRESS AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('CHANGER SA RELATION À L'argent avec le Coaching Mental') "; expected "[, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values ('Académie d\'Été');
insert into formation ( intitule ) values ('Académie d’Hiver');
insert into formation ( intitule ) values ('L’Art des Métaphores dans le Coaching Mental');
insert into formation ( intitule ) values ('Auto-hypnose');
insert into formation ( intitule ) values ('Renforcer la Confiance en Soi');
insert into formation ( intitule ) values ('Changer vos Croyances Limitantes avec le Coaching Mental');
insert into formation ( intitule ) values ('Comment Lancer votre Activité de Coach Mental');
insert into formation ( intitule ) values ('Apprendre la Sophrologie');
insert into formation ( intitule ) values ('Comprendre et Intégrer les Emotions');
insert into formation ( intitule ) values ('Maigrir avec le Coaching Mental');
insert into formation ( intitule ) values ('Techniques de Coaching Mental pour le Sport');
insert into formation ( intitule ) values ('Gérer le Stress avec le Coaching Mental');
insert into formation ( intitule ) values ('Changer sa relation à l'argent avec le Coaching Mental') [42001-170]
03-06 14:55:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Column "Académie d'Été" not found; SQL statement:
insert into formation ( intitule ) values ("Académie d'Été") [42122-170]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:141)
	at org.h2.command.dml.Insert.prepare(Insert.java:231)
	at org.h2.command.Parser.prepareCommand(Parser.java:218)
	at org.h2.engine.Session.prepareLocal(Session.java:414)
	at org.h2.engine.Session.prepareCommand(Session.java:363)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:164)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:152)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1311)
	at org.h2.server.web.WebApp.query(WebApp.java:1001)
	at org.h2.server.web.WebApp$1.next(WebApp.java:964)
	at org.h2.server.web.WebApp$1.next(WebApp.java:953)
	at org.h2.server.web.WebThread.process(WebThread.java:166)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
03-06 14:55:16 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "insert into formation ( intitule ) values ('Changer sa relation à l'argent avec le Coaching Mental');
insert into formation ( intitule ) values ('Décoder le Non-Verbal pour Mieux s’Accorder (2 jours)');
insert into formation ( intitule ) values ('Décoder les Gestes de la Séduction (2 jours)');
insert into formation ( intitule ) values ('Les Clés de la Communication d’Influence (2 jours)[*]');"; SQL statement:
insert into formation ( intitule ) values ('Changer sa relation à l'argent avec le Coaching Mental');
insert into formation ( intitule ) values ('Décoder le Non-Verbal pour Mieux s’Accorder (2 jours)');
insert into formation ( intitule ) values ('Décoder les Gestes de la Séduction (2 jours)');
insert into formation ( intitule ) values ('Les Clés de la Communication d’Influence (2 jours)'); [42000-170]
03-06 14:55:55 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "INSERT INTO FORMATION ( INTITULE ) VALUES ('Académie d'ÉTÉ[*]');
insert into formation ( intitule ) values ('ACADÉMIE D’HIVER');
insert into formation ( intitule ) values ('L’ART DES MÉTAPHORES DANS LE COACHING MENTAL');
insert into formation ( intitule ) values ('AUTO-HYPNOSE');
insert into formation ( intitule ) values ('RENFORCER LA CONFIANCE EN SOI');
insert into formation ( intitule ) values ('CHANGER VOS CROYANCES LIMITANTES AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('COMMENT LANCER VOTRE ACTIVITÉ DE COACH MENTAL');
insert into formation ( intitule ) values ('APPRENDRE LA SOPHROLOGIE');
insert into formation ( intitule ) values ('COMPRENDRE ET INTÉGRER LES EMOTIONS');
insert into formation ( intitule ) values ('MAIGRIR AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('TECHNIQUES DE COACHING MENTAL POUR LE SPORT');
insert into formation ( intitule ) values ('GÉRER LE STRESS AVEC LE COACHING MENTAL');
insert into formation ( intitule ) values ('CHANGER SA RELATION À L'argent avec le Coaching Mental') "; expected "[, ::, *, /, %, +, -, ||, ~, !~, NOT, LIKE, REGEXP, IS, IN, BETWEEN, AND, OR, ,, )"; SQL statement:
insert into formation ( intitule ) values ('Académie d'Été');
insert into formation ( intitule ) values ('Académie d’Hiver');
insert into formation ( intitule ) values ('L’Art des Métaphores dans le Coaching Mental');
insert into formation ( intitule ) values ('Auto-hypnose');
insert into formation ( intitule ) values ('Renforcer la Confiance en Soi');
insert into formation ( intitule ) values ('Changer vos Croyances Limitantes avec le Coaching Mental');
insert into formation ( intitule ) values ('Comment Lancer votre Activité de Coach Mental');
insert into formation ( intitule ) values ('Apprendre la Sophrologie');
insert into formation ( intitule ) values ('Comprendre et Intégrer les Emotions');
insert into formation ( intitule ) values ('Maigrir avec le Coaching Mental');
insert into formation ( intitule ) values ('Techniques de Coaching Mental pour le Sport');
insert into formation ( intitule ) values ('Gérer le Stress avec le Coaching Mental');
insert into formation ( intitule ) values ('Changer sa relation à l'argent avec le Coaching Mental') [42001-170]
03-07 19:35:00 jdbc[2]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "UPDATE MEMBRE SET IDCONTACT[*] ! -1 "; SQL statement:
update membre set idcontact ! -1 [42000-170]
